config/config.go

```diff
@@ -167,27 +167,27 @@ type Coordinator struct {
 		// been forged or marked as invalid for longer than the
 		// SafetyPeriod and pending L2Txs that have been in the pool
 		// for longer than TTL once there are MaxTxs.
-		PurgeBlockDelay int64 `validate:"required"`
+		PurgeBlockDelay int64 `validate:"required,gte=0"`
 		// InvalidateBlockDelay is the delay between blocks to mark
 		// invalid transactions due to nonce lower than the account
 		// nonce.
-		InvalidateBlockDelay int64 `validate:"required"`
+		InvalidateBlockDelay int64 `validate:"required,gte=0"`
 	} `validate:"required"`
 	TxSelector struct {
 		// Path where the TxSelector StateDB is stored
-		Path string `validate:"required"`
+		Path string `validate:"required,file"`
```

Comment:
The `file` requirement seems to check that the string is an existing file: https://github.com/go-playground/validator/blob/b926bf0df97108effd04f9325365cd7969935e16/baked_in.go#L1289
But in this case we have a different requirement
1. It's not a file, it's a path
2. The current implementation doesn't require the path to exists, it will create it if it doesn't exists.  The real requirement would be that the path can be created and written into by the user running the node.  The same applies to the rest of the fields called `Path` that have the constraint `file`.